hbase-server/src/test/java/org/apache/hadoop/hbase/TestClientClusterMetrics.java

```diff
@@ -72,6 +84,8 @@
   public static void setUpBeforeClass() throws Exception {
     Configuration conf = HBaseConfiguration.create();
     conf.set(CoprocessorHost.MASTER_COPROCESSOR_CONF_KEY, MyObserver.class.getName());
+    conf.set("hbase.security.authorization", Boolean.TRUE.toString());
+    conf.set("hbase.security.exec.permission.checks", Boolean.TRUE.toString());
```

Comment:
If you're setting these, do you also want to deploy the AccessController?

If not, what are these needed for?

Reply:
Removed them as they were added when I was testing something else.